kDrive/UI/Controller/Files/File List/FileListViewModel.swift

```diff
@@ -342,7 +342,10 @@ class FileListViewModel: SelectDelegate {
     }
 
     func didSelectFile(at indexPath: IndexPath) {
-        guard let file: File = getFile(at: indexPath) else { return }
+        guard let file: File = getFile(at: indexPath) else {
+            return
+        }
+
```

Comment:
Out of scope for the PR
```suggestion
        guard let file: File = getFile(at: indexPath) else { return }
```

Reply:
If this is the way we should do it, it should be done by the linter IMO.